casper/src/main/scala/coop/rchain/casper/Validate.scala

```diff
@@ -58,13 +58,15 @@ object Validate {
         if verifySig(candidateBytesDigest, signature.sig.toByteArray, publicKey.toByteArray)
       } yield publicKey).toSet
 
+    val logMsg = signatories.isEmpty match {
+      case true => s"ApprovedBlock is self-signed by ceremony master."
+      case false =>
+        s"ApprovedBlock is signed by: ${signatories
+          .map(x => "<" + Base16.encode(x.toByteArray).substring(0, 10) + "...>")
```

Comment:
I think we have `Pretty.buildString`.

Reply:
I believe I did not touch this line, and don't see such helper in PrettyPrinter